### PR TITLE
Bugfix/icons for multiselect detail

### PR DESCRIPTION
**Pull request by flourish86**

### CODE REVIEW

**Comment by nilmerg** [comment]
Please use master as base for this branch, otherwise I can't review this without having to look at all the changes from #206 again.

**Comment by flourish86** [comment]
> Please use master as base for this branch, otherwise I can't review this without having to look at all the changes from #206 again.

Sorry, my fault. fixed it and also the CodeSniffer error.

File: library/Icingadb/Widget/Detail/MultiselectQuickActions.php [hunk 1]
```diff
@@ -158,7 +159,7 @@ protected function assemble()
             $this->assembleAction(
                 'processCheckresult',
                 t('Process check result'),
-                'icon-edit',
+                'random',
```
Comment (nilmerg) [inline library/Icingadb/Widget/Detail/MultiselectQuickActions.php:162]: Isn't that the icon that's also being used for indicating a flapping host/service? (If not exactly this icon, it's at least a very similar one)

Personally, I'd leave this the same as before, but change the icon for the acknowledge action. (Maybe the check icon or so)

Comment (flourish86) [reply library/Icingadb/Widget/Detail/MultiselectQuickActions.php:162]: Sounds good. I'll change it.

Comment (flourish86) [reply library/Icingadb/Widget/Detail/MultiselectQuickActions.php]: ✅

**Review by nilmerg — APPROVED** [review]

**Pull request merged**